packages/api/package.json

```diff
@@ -48,12 +50,13 @@
     "@web3-storage/multipart-parser": "^1.0.0",
     "itty-router": "^2.3.10",
     "multiformats": "^9.0.4",
-    "p-retry": "^4.6.1"
+    "p-retry": "^4.6.1",
+    "uint8arrays": "^3.0.0"
   },
   "bundlesize": [
     {
       "path": "./dist/main.js",
-      "maxSize": "1 MB",
+      "maxSize": "1.4 MB",
```

Comment:
This makes a huge difference given https://bundlephobia.com/package/@aws-sdk/client-s3@3.28.0 

As this is the worker, it is not problematic, even though it is a lot

Reply:
bundlephobia says it tree-shakes, so may not be so bad. is our build process taking advantage of that?

Reply:
oh, I was not aware of it. Per the docs:

> A Workers script can be up to 1MB in size after compression.

our bundlesize check is done pre-compression though. I will look into that too

Reply:
ok, making the bundle size job test compressed made the bundle size change from `1.06MB` to `287.12KB`